ballerina/Ballerina.toml

```diff
@@ -6,7 +6,7 @@ version = "0.1.0"
 license = ["Apache-2.0"]
 authors = ["Ballerina"]
 keywords = []
```

Comment:
Shall we add some keywords here?
There is already an older connector for Dropbox. You can check for the keywords there. And may add some others if necessary
https://central.ballerina.io/ballerinax/dropbox/1.5.1

Reply:
Done